exporter/prometheusexporter/config.go

```diff
@@ -46,6 +46,9 @@ type Config struct {
 	// ResourceToTelemetrySettings defines configuration for converting resource attributes to metric labels.
 	ResourceToTelemetrySettings resourcetotelemetry.Settings `mapstructure:"resource_to_telemetry_conversion"`
 
+	// EnableOpenMetrics enables the use of the OpenMetrics encoding option for the prometheus exporter.
+	EnableOpenMetrics bool `mapstructure:"enable_open_metrics"`
```

Comment:
Wondering if this should be inside a section for handler options, so that we can namespace this

Reply:
I thought requiring a different export format per namespace would be a rather rare use case and so making this an exporter level config made more sense.